### ISSUE TITLE
Upgrading armour doesn't work if you have 0 bars

### ISSUE THREAD

**Issue by mrDLSable** [MelvorIdle/melvoridle.github.io#21]
**Describe the bug**
If you have 0 bars of the next upgrade in your inventory, the upgrade UI doesn't open when clicking the upgrade button

**To Reproduce**
1. Have a piece of armour and no bars of the next trim in your inventory.
2. Press the upgrade button

**Expected behavior**
The upgrade UI should appear

**Screenshots**
![image](https://user-images.githubusercontent.com/1783781/68702466-29e8c300-0589-11ea-9de9-7e97ab0f460a.png)

**Browser**
Chrome

**Console output**
```
Melvor Cloud Connection: Failed
save.js?91:40 Loading Save Game...
save.js?91:171 Save Game Loaded...
save.js?91:40 Loading Save Game...
save.js?91:131 Updating bank slots to new formula
save.js?91:142 0 GP refunded for bank slot adjustment
save.js?91:147 Updating bank slots to new new formula
save.js?91:159 No refund required for new bankslot formula
save.js?91:171 Save Game Loaded...
smithing2.js?91:194 Loading Smithing...
mastery.js?91:167 Updating Masteries
mastery.js?91:35 Loading Masteries...
addItemToBank(CONSTANTS.item.Iron_Boots, 1)
true
2bank.js?91:809 Uncaught TypeError: Cannot read property 'qty' of undefined
    at upgradeItem (bank.js?91:809)
    at HTMLButtonElement.onclick ((index):1)
upgradeItem @ bank.js?91:809
onclick @ (index):1
addItemToBank(CONSTANTS.item.Silver_Bar, 1)
true
```

**Comment by mrDLSable** [comment]
Having 1 or more of the trim ingots in your inventory allows the UI to open.

**Comment by mrDLSable** [comment]
`if (checkBankForItem >= 0) {` allows code execution if checkBankForItem is false, because false == 0.

**Comment by MelvorIdle** [comment]
Fixed in v0.09.2
